lib/mutant.rb

```diff
@@ -204,7 +204,11 @@ class Config
       integration:       Integration::Null,
       isolation:         Mutant::Isolation::Fork,
       jobs:              Mutant.ci? ? CI_DEFAULT_PROCESSOR_COUNT : ::Parallel.processor_count,
+      kernel:            Kernel,
```

Comment:
So you've audited the code to make sure it doesn't refer to anything outside of the `Mutant` namespace, besides here of course?